packages/template-retail-react-app/app/pages/checkout/index.jsx

```diff
@@ -59,9 +59,9 @@ const Checkout = () => {
                             )}
 
                             <ContactInfo />
-                            <ShippingAddress />
-                            <ShippingOptions />
-                            <Payment />
+                            {/* <ShippingAddress /> */}
+                            {/* <ShippingOptions /> */}
+                            {/* <Payment /> */}
```

Comment:
I made changes to shared checkout context broke the sections, I'm going to just comment them out and there are tickets to follow up on each of the sections.